docs/Configuration.md

```diff
@@ -87,6 +87,62 @@ data:
   NAMESPACES: "default"    # ""
 ```
 
+### Authentication to Remote Services
+
+Pelorus exporters make use of `personal access tokens` when authentication is 
+required.  It is recommended to configure the Pelorus exporters with authenticaion
+via the `TOKEN` key to avoid connection rate limiting and access restrictions.
+
+More information about personal access tokens:
+* [Github Personal Access Tokens](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)
+
+* [Jira / Bitbucket Personal Access Tokens](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
+
+* [Gitlab Personal Access Tokens](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+
+* [Microsoft Azure DevOps Tokens](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows)
+
+* [Gitea Tokens](https://docs.gitea.io/en-us/api-usage/#generating-and-listing-api-tokens)
+
+To store a personal access token securely and to make the token available to Pelorus use
+Openshift secrets. Pelorus can utilize secrets for all of the exporters. The following
+is an example for the committime exporter.
+
+A simple Github example:
+```shell
+oc create secret generic github-secret --from-literal=TOKEN=<personal access token> -n pelorus
```

Comment:
I do wonder if we should also mention possibility of using example configs to create secrets rather than passing them on the command line only, see:

https://github.com/konveyor/pelorus/tree/master/charts/pelorus/secrets

Reply:
I think I agree.. we should be pushing people towards the config maps :)  Will update in a bit

Reply:
It's not config map, it's secret which is defined in a file, but similar to config map.

Reply:
oh ya.. sorry read that too quickly..
cli vs. template.. six of one, half dozen of the other.. worth mentioning though.

Reply:
added